#4204 Updated for server details to not show for mitm.it

Samhita-sopho opened the pull request:
#### Description

Server section does not render if server address is null ( as seen in mitmproxy ) . Added a tflow with null server address to check if that fixes the code coverage issue as in the previous PR

#### Checklist

 - [x] I have updated tests where applicable.
 - [x] I have added an entry to the CHANGELOG.

Kriechi commented:
I think you are missing some commit in this PR - I only see the updated changelog but no actual code changes...
ref #4204

Kriechi commented on CHANGELOG:
```diff
@@ -2,6 +2,7 @@ Unreleased: mitmproxy next
 
 ** Full Changelog **
 
+    * UI appears blank in mitmweb for mitm.it flow(@samhita-sopho)
```
Please rephrase to something "that you fixed".
"fixed xyz" or "added feature xyz"

Samhita-sopho replied:
Thank you for pointing it , after updating the same I found that the test for web ui fails so working on it

Samhita-sopho commented:
@Kriechi I have tested with npm test on my local machine and have seen this output 
Test Suites: 59 passed, 59 total
Tests:       316 passed, 316 total
Snapshots:   103 passed, 103 total
Time:        4.856s , 
**Summary of changes**: Made a null check to details.jsx and added a test flow with different parameters(_tflowServerAddressNull.js) , added a snapshot to test against and updated detailsSpec.js as well. Could you please let me know there is something else that needs to be done here

mhils commented:
This seems to be a somewhat chaotic duplicate of #4224?

Closing this here for now.
